google-cloud-storage/src/main/resources/reference.conf

```diff
@@ -1,6 +1,22 @@
-alpakka.google.cloud.storage {
-  base-url = "https://www.googleapis.com/" // default
-  base-path = "/storage/v1" // default
-  token-url = "https://www.googleapis.com/oauth2/v4/token" // default
-  token-scope = "https://www.googleapis.com/auth/devstorage.read_write" // default
+alpakka.google {
+
+  credentials {
+    default-scopes = ${?alpakka.google.scopes} ["https://www.googleapis.com/auth/devstorage.read_write"]
+  }
+
+  cloud.storage {
+
+    # Deprecated, use config path alpakka.google.credentials
+    project-id = deprecated
+    client-email = deprecated
+    private-key = deprecated
+
+    # No longer supported, use config path alpakka.google.forward-proxy
```

Comment:
These are the configurable endpoints I dropped support for.

Reply:
I don't know. Maybe some people with some recent experience using GCS can comment. @francisdb, @DanieleSassoli, @gabrielreid ?

See Arman's note about introducing configuration breaking changes here: https://github.com/akka/alpakka/pull/2613#issuecomment-802101303

Reply:
Just had a look, seems this was added by @jkobejs when he worked further on my pr. We are not using these settings so for my part ok to make unsupported.